github/repos.go

```diff
@@ -1321,3 +1321,31 @@ func (s *RepositoriesService) Transfer(ctx context.Context, owner, repo string,
 
 	return r, resp, nil
 }
+
+// DispatchRequest represents a request to trigger a repository_dispatch event.
+type DispatchRequest struct {
```

Comment:
Let's please call this `DispatchRequestOptions` for consistency with the repo.

Reply:
Hi @fleskesvor, it looks like this comment was not addressed. Could you please fix this?

Reply:
Oh, right. I actually did that before, but changed it to `DispatchRequest` after looking at the functiondirectly above. I will change it back again.